lessons/operator/02/README.md

```diff
@@ -1,257 +1,267 @@
+      ```
+
+      The output should indicate that all of the services are `Up (healthy)` or "completed" (`Exit 0`).
+
+      **Example Output:**
+
+      ```shell
+                   Name                        Command                       State                                                         Ports
+      --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+      workshop_configurator_1    seed-workshop-resources          Exit 0
+      workshop_grafana_1         /run.sh                          Up (healthy)            0.0.0.0:3001->3000/tcp
+      workshop_influxdb_1        /entrypoint.sh influxd           Up (healthy)            0.0.0.0:8086->8086/tcp
+      workshop_sensu-agent_1     sensu-agent start --log-le ...   Up (healthy)            2379/tcp, 2380/tcp, 3000/tcp, 0.0.0.0:49221->3031/tcp, 8080/tcp, 8081/tcp, 0.0.0.0:49220->8125/udp
+      workshop_sensu-backend_1   sensu-backend start --log- ...   Up (healthy)            2379/tcp, 2380/tcp, 0.0.0.0:3000->3000/tcp, 0.0.0.0:8080->8080/tcp, 0.0.0.0:8081->8081/tcp
+      workshop_sensuctl_1        wait-for-sensu-backend sen ...   Exit 0
+      workshop_vault_1           docker-entrypoint.sh vault ...   Up (healthy)            0.0.0.0:8200->8200/tcp
+      ```
+
+      Every container should show a status of `Up (healthy)` or `Exit 0`.
+      
+      If any containers have the status `Up` or `Up (health: starting)`, wait a few seconds then retry the `sudo docker-compose ps` command.
+
+      > **TROUBLESHOOTING:**
+      > 
+      > If any containers have completed with the `Exit 1` or `Exit 2` state, it's possible that these were the result of an intermittent failure (e.g. if the `sensu-backend` container was slow to start). Re-running the `sudo docker-compose up -d` command will resolve the issue.
+
+   1. **Check that the Sensu Web UI is Running**
+
+      The Sensu Web UI should be available at [http://127.0.0.1:3000](http://127.0.0.1:3000).
+      Login with the user `sensu` and password `sensu`.
+   
+   1. **Check that Rocket.Chat is Running**
+
+      The Rocket.Chat app should be available at [http://127.0.0.1:5000](http://127.0.0.1:5000).
+      Login with the user `sensu` and password `sensu`.
+
+
+**NEXT:** If all the containers show as `Up (healthy)` or `Exit 0` state, and you can reach the [Sensu Web UI](http://127.0.0.1:3000) and [Rocket.Chat](http://127.0.0.1:5000) apps, then you're ready to start the workshop!
+
+## Discussion
+
+In this lesson you installed a local sandbox environment, and verified that it was up and running. 
+You learned about the various components of a Sensu-enabled monitoring environment and how they work together.
+
+In the following lessons, we will install the `sensuctl` command-line tool, and learn how to do some essential tasks via hands-on exercises using this environment. 
+
+### Troubleshooting Guide
```

Comment:
I love this section. Great job! 

One additional subheading that might be helpful here could be: 

> #### Need to stop the workshop environment and resume later? 
>
> No problem! The Sensu Go Workshop can be stopped without losing your progress by running the following command: 
> 
> **Mac and Linux:** 
> 
> ```
> sudo docker-compose down
> ```
> 
> **Windows (Powershell):**
> 
> ```
> docker-compose down
> ```
> 
> _NOTE: running `docker-compose down` **without** the `-v` flag instructs Docker to stop all of the running containers but leave the Docker volumes intact._

Reply:
Nice one!